### DISCUSSION TITLE
Hide/show fields on multiple pages

### DISCUSSION THREAD

**Discussion by MMGIFL** [filamentphp/filament#2972] [Ideas]
Would like to be able to include multiple pages on the hiddenOn()/visibleOn() attributes, this would allow multiple custom pages to have  or not have certain form fields or table views allowing a much more complex layout of custom pages. 

`->hiddenOn([Pages\EditProject::class, Pages\CustomProject::class])`

**Comment by ryangjchandler** [comment]
https://github.com/filamentphp/filament/pull/2973
